File: src/tool/spring-boot-generator/output/contract/contract/src/main/java/com/contract/pojo/entity/ContractSign.java
package com.contract.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("")
@Getter
@Setter
public class ContractSign  {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("签约编号")
    private Integer id;

    @ApiModelProperty("需要付费")
    private Integer needPay;

    @ApiModelProperty("通知次数")
    private Integer notifyCount;

    @ApiModelProperty("通知邮箱")
    private String notifyEmail;

    @ApiModelProperty("通知号码")
    private String notifyPhone;

    @ApiModelProperty("私信内容")
    private String shortMsg;

    @ApiModelProperty("签署IP")
    private String signIp;

    @ApiModelProperty("签署人")
    private String signPerson;

    @ApiModelProperty("签署顺序")
    private Integer signSort;

    @ApiModelProperty("签署状态")
    private Integer signStatus;

    @ApiModelProperty("签署主体")
    private String signSubject;

    @ApiModelProperty("签署时间")
    private Date signTime;

    @ApiModelProperty("签署URL")
    private String signUrl;

    @ApiModelProperty("主体类型")
    private Integer subjectType;

    @ApiModelProperty("用户编号")
    private Integer userId;

}

File: src/tool/spring-boot-generator/output/contract/db/contractSign.js
let contractSign = {
	name: "contractSign",
	description: "",
	columns: {
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		id: {
			name: "id",
			description: "签约编号",
			isPk: true,
			autoIncrement: true,
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		needPay: {
			name: "needPay",
			description: "需要付费",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		notifyCount: {
			name: "notifyCount",
			description: "通知次数",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		notifyEmail: {
			name: "notifyEmail",
			description: "通知邮箱",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		notifyPhone: {
			name: "notifyPhone",
			description: "通知号码",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		shortMsg: {
			name: "shortMsg",
			description: "私信内容",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signIp: {
			name: "signIp",
			description: "签署IP",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signPerson: {
			name: "signPerson",
			description: "签署人",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signSort: {
			name: "signSort",
			description: "签署顺序",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		signStatus: {
			name: "signStatus",
			description: "签署状态",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		signSubject: {
			name: "signSubject",
			description: "签署主体",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		signTime: {
			name: "signTime",
			description: "签署时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		signUrl: {
			name: "signUrl",
			description: "签署URL",
			nullable: true,
			type: {
				name: "varchar",
				length: null
			}
		},
		subjectType: {
			name: "subjectType",
			description: "主体类型",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		userId: {
			name: "userId",
			description: "用户编号",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"createTime",
	"id",
	"needPay",
	"notifyCount",
	"notifyEmail",
	"notifyPhone",
	"shortMsg",
	"signIp",
	"signPerson",
	"signSort",
	"signStatus",
	"signSubject",
	"signTime",
	"signUrl",
	"subjectType",
	"userId"
];

function create() {
  return {};
}

contractSign.columnsArray=columnsArray;
contractSign.create=create;
module.exports=contractSign;